documentation/dev/setup-and-guidelines.rst

```diff
@@ -166,6 +166,18 @@ A rolling log file handler is used, so if ``flexmeasures.log`` gets to a few meg
 The default logging level is ``WARNING``. To see more, you can update this with the config setting ``LOGGING_LEVEL``, e.g. to ``INFO`` or ``DEBUG``
 
 
+Mocking an Email Server for Development
+--------------------------------
+
+To handle emails locally during development, you can use MailHog. Follow these steps to set it up:
+
+.. code-block:: bash
+
+   $ docker run -p 8025:8025 -p 1025:1025 --name mailhog mailhog/mailhog
+   $ export MAIL_PORT=1025  # Add this to your local flexmeasures.cfg
```

Comment:
I would say "You can also add this ..."

Reply:
do you mean before the code block it should say You can also add this, to handle emails locally during development:

Reply:
or i think you mean to in the comment on line 177

Reply:
Yes just before the comment